config.go

```diff
@@ -22,7 +22,7 @@ type config struct {
 	Trigger     string            `yaml:"trigger"`
 	Builder     string            `yaml:"builder"`
 	BuilderMap  map[string]string `yaml:"builderMap"`
-	EnvVars     map[string]string `yaml:"envVars"`
+	Env         map[string]string `yaml:"env"`
```

Comment:
@lance won't this cause issues for users using newer versions with older conf file?

Reply:
But not many people use `function` so it is not that much of a problem. We just need to let know people that it's been changed.

Reply:
Yes - you are right that any existing users would need to change their config file. But this is all so new, and it's still unsupported, so I think perhaps adding to release notes for TP-2 would be OK instead of trying for backwards compatibility. But I will follow up on this to be sure it's OK.

Reply:
LGTM!

Yes, this is technically a backwards incompatible change, but let's get this thing squeaky-clean now, before 1.0, while we still can.  (I also doubt there are others using env vars other than we team members at this point?),